rust-bot/src/main.rs

```diff
@@ -113,7 +115,20 @@ fn pin(ctx: &mut Context, msg: &Message, mut args: Args) -> CommandResult {
         error!(err = ?e);
         println!("{}", e);
     }
-    
+
+    Ok(())
+}
+
+#[command]
+#[aliases("yee-claw")]
+fn yee_claw(ctx: &mut Context, msg: &Message) -> CommandResult {
```

Comment:
[Optional]: If you want, we can add tracing to this command. Just need to add an `#[instrument]` and register the root. We can then pipe the error to Honeycomb with the `err!` macro via `err!(err = ?why)`

```suggestion
#[instrument(skip(ctx))]
fn yee_claw(ctx: &mut Context, msg: &Message) -> CommandResult {
let _result = register_dist_tracing_root(generate_trace_id_from_message(msg), None);
```

Reply:
I dig all of this! I meant to keep this but must have lost it in my rebase, thank you for the suggestion!